Content.Server/Bodycamera/BodyCameraSystem.cs

```diff
@@ -0,0 +1,131 @@
+using Content.Server.Access.Systems;
+using Content.Server.Popups;
+using Content.Server.PowerCell;
+using Content.Server.SurveillanceCamera;
+using Content.Shared.Clothing.Components;
+using Content.Shared.Examine;
+using Content.Shared.Inventory.Events;
+using Content.Shared.PowerCell;
+using Content.Shared.PowerCell.Components;
+using Content.Shared.Bodycamera;
+using Content.Shared.Timing;
+using Robust.Shared.Containers;
+
+namespace Content.Server.Bodycamera;
+
+public sealed class BodyCameraSystem : SharedBodyCameraSystem
+{
+    [Dependency] private readonly SurveillanceCameraSystem _camera = default!;
+    [Dependency] private readonly IdCardSystem _idCardSystem = default!;
+    [Dependency] private readonly SharedContainerSystem _containerSystem = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<BodyCameraComponent, PowerCellSlotEmptyEvent>(OnPowerCellSlotEmpty);
+        SubscribeLocalEvent<BodyCameraComponent, PowerCellChangedEvent>(OnPowerCellChanged);
+    }
+
+    /// <summary>
+    /// Sync the SurveillanceCameraComponent state (default enabled) to the BodyCamera state (default disabled)
+    /// </summary>
+    protected override void OnComponentStartup(EntityUid uid, BodyCameraComponent component, ComponentStartup args)
+    {
+        base.OnComponentStartup(uid, component, args);
+
+        if (HasComp<SurveillanceCameraComponent>(uid))
+            _camera.SetActive(uid, false);
+    }
```

Comment:
Use TryComp and pass it in because resolving it twice is just going to be twice as expensive.